### PR TITLE
Origin/bael 4785

### PR DESCRIPTION
**Pull request by vishal1023**

### CODE REVIEW

File: spring-core-5/src/main/java/com/baeldung/springbean/naming/service/MessagingServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.baeldung.springbean.naming.service;
+
+import com.baeldung.springbean.naming.component.MyCustomComponent;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MessagingServiceImpl implements MessagingService{
```
Comment (JonCook) [inline spring-core-5/src/main/java/com/baeldung/springbean/naming/service/MessagingServiceImpl.java:9]: minor formatting issue here with the opening parenthesis after implements

File: spring-core-5/src/main/java/com/baeldung/springbean/naming/component/Cat.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.baeldung.springbean.naming.component;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+@Qualifier("cat")
+public class Cat implements Animal{
```
Comment (JonCook) [inline spring-core-5/src/main/java/com/baeldung/springbean/naming/component/Cat.java:8]: minor formatting issue here with the opening parenthesis after implements

File: spring-core-5/src/test/java/com/baeldung/springbean/naming/SpringBeanNamingUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    private AnnotationConfigApplicationContext context;
+
+    @BeforeEach
+    void setUp() {
+        context = new AnnotationConfigApplicationContext();
+        context.scan("com.baeldung.springbean.naming");
+        context.refresh();
+    }
+
+    //    To name a bean spring gets the class name and converts the first letter to lowercase.
```
Comment (JonCook) [inline spring-core-5/src/test/java/com/baeldung/springbean/naming/SpringBeanNamingUnitTest.java:25]: Minor comment on the formatting. No need for tabs in the comments. One space is enough. See our own eclipse formatter for example - https://github.com/eugenp/tutorials/blob/master/eclipse/formatter.xml

**Review by JonCook — CHANGES_REQUESTED** [review]
Hi @vishal1023 

Just a few very minor formatting issues here to fix.

Thanks.

File: spring-core-5/src/main/java/com/baeldung/springbean/naming/service/MessagingServiceImpl.java (hunk 1 above)
Comment (vishal1023) [reply spring-core-5/src/main/java/com/baeldung/springbean/naming/service/MessagingServiceImpl.java]: fixed

File: spring-core-5/src/main/java/com/baeldung/springbean/naming/component/Cat.java (hunk 2 above)
Comment (vishal1023) [reply spring-core-5/src/main/java/com/baeldung/springbean/naming/component/Cat.java]: fixex

File: spring-core-5/src/test/java/com/baeldung/springbean/naming/SpringBeanNamingUnitTest.java (hunk 3 above)
Comment (vishal1023) [reply spring-core-5/src/test/java/com/baeldung/springbean/naming/SpringBeanNamingUnitTest.java]: fixed

**Pull request merged**